### ISSUE TITLE
Control over GPU fans only applied after selecting "Match fan automatically..."

### ISSUE THREAD

**Issue by huhwhat** [Rem0o/FanControl.Releases#94]
I have a Gigabyte 1660 Ti Gaming OC 6G that cannot be controlled by FanControl until "Match fan automatically..." is selected. Matching the fan manually doesn't work. Toggling the fan control with the green button doesn't work. I've tried restarting the computer with MSI Afterburner turned off but the same thing still happens so I have to match fan automatically every time my computer starts up to apply FanControl settings. Control is automatically lost too when I adjust the fan curve in FanControl so I need to match fan automatically every time I make any changes. Incidentally, I've had no luck using MSI Afterburner in controlling my GPU's fan speed at all, which is why I turned to FanControl.

This doesn't happen with the control I set for my case fans using FanControl. Possible interference from my GPU's BIOS? Is anyone else with the same GPU or a GPU from the same company experiencing the same issue?

**Comment by Rem0o** [comment]
"Match automatically" doesn't do any magic. It simply toggles the fan to 60%, then down, then back to whatever is was before. If AfterBurner also doesn't work, your GPU bios must indeed not be supported or have a setting that prevents it from being externally controlled. There's nothing really I can do here.

**Comment by huhwhat** [comment]
Thanks for the comment. It's my fault for not noticing that Afterburner does indeed work in controlling fan speed. The wrong impression I had of fan speed not being controlled by Afterburner was due to the fan speed jumping constantly between 0 RPM to 1000 RPM, something to do with the fan start and stop percentages. Issue is resolved if I set the fan speed above fan start percentage.

Disabling fan control in Afterburner doesn't take effect immediately and I have to use "match automatically" to let FanControl have control over the fan again, which made me think that it's magic. 😛
